static/src/js/components/Panels/PanelGroupHeader.jsx

```diff
@@ -311,9 +320,8 @@ export const PanelGroupHeader = ({
                       <RadioSettingDropdown
                         id={`panel-group-header-dropdown__sort__${panelGroupId}`}
                         className="panel-group-header__setting-dropdown"
-                        activeSortOrder={activeSort}
```

Comment:
I'm not sure why this was set to this its not a prop or being used for this component